components/homekit/esp_hap_apple_profiles/include/hap_apple_chars.h

```diff
@@ -142,6 +142,13 @@ extern "C" {
 #define HAP_CHAR_UUID_REMAINING_DURATION                        "D4"
 #define HAP_CHAR_UUID_VALVE_TYPE                                "D5"
 #define HAP_CHAR_UUID_IS_CONFIGURED                             "D6"
+#define HAP_CHAR_UUID_SUPPORTED_VIDEO_STREAM_CONFIGURATION		 "114"
+#define HAP_CHAR_UUID_SUPPORTED_AUDIO_CONFIGURATION				 "115"
+#define HAP_CHAR_UUID_SUPPORTED_RTP_CONFIGURATION					 "116"
+#define HAP_CHAR_UUID_SELECTED_RTP_STREAM_CONFIGURATION			 "117"
+#define HAP_CHAR_UUID_SETUP_ENDPOINTS									 "118"
+#define HAP_CHAR_UUID_STREAMING_STATUS									 "120"
```

Comment:
Looks like you might have an alignment issue here.

Reply:
Yeah as noted in the original comment, I've mistakenly auto formatted the files. I'll revert this at a later point.

Really looking for thoughts and feedback about the approach and how I should represent TLV8 configs for the video stuff.